src/Query/BaseBuilder.php

```diff
@@ -179,11 +179,7 @@ public function select(...$columns)
      */
     public function getCountQuery()
     {
-        $without = ['columns' => [], 'limit' => null];
-
-        if (empty($this->groups)) {
-            $without['orders'] = [];
-        }
```

Comment:
This is the only code change, the rest was generated by applying the diff supplied by Style CI.